resources/lang/pt-br/forum.php

```diff
@@ -19,103 +19,98 @@
  */
 
 return [
-    'pinned_topics' => 'Tópicos Fixados',
-    'subforums' => 'Sub-fórums',
-    'title' => 'comunidade osu!',
 
     'covers' => [
         'create' => [
             '_' => 'Definir imagem de capa',
             'button' => 'Enviar imagem',
-            'info' => 'O tamanho da capa deve ser :dimensions. Você também pode arrastar sua imagem aqui para enviar.',
+            'info' => 'O tamanho da capa deve ser :dimensions. Você também pode soltar sua imagem aqui para fazer enviar.',
```

Comment:
"fazer" does not seem to be a good pronunciation.
"arrastar" is more used in another websites.